.github/workflows/qc.yml

```diff
@@ -9,6 +9,8 @@ on:
     branches: [ master ]
   pull_request:
     branches: [ master ]
+  pull_request_target:
+    branches: [ master ]
```

Comment:
Given the huge list of `activity types` listed in:

https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request_target

Isnt it a bit risky to not write them out? Like: editing the PR text? adding a label?

Maybe add something like:

```
types: [assigned, opened, synchronize, reopened]
```

Reply:
No need to add this in the end.